apax/train/callbacks.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+import tensorflow as tf
+from keras.callbacks import CSVLogger, TensorBoard
+
+log = logging.getLogger(__name__)
+
+
+def initialize_callbacks(callback_configs, model_version_path):
+    log.info("Initializing Callbacks")
+
+    dummy_model = tf.keras.Model()
+    callback_dict = {
+        "csv": {
+            "class": CSVLogger,
+            "log_path": model_version_path / "log.csv",
+            "path_arg_name": "filename",
+            "kwargs": {"append": True},
+            "model": dummy_model,
```

Comment:
Could be a datalcass using

```py

@dataclass
class Callback:
    class: object
    log_path: Path
    path_arg_name: str
    kwargs: dict
    model: object

# and
dataclasses.asdict(...)
```

if more loggers are supposed to be added in the future.